content/get-haiku/release-notes-r1b2.md

```diff
@@ -0,0 +1,271 @@
+ - [#13808](https://dev.haiku-os.org/ticket/13808) Overall, it is more clear when HaikuDepot is contacting the web to retrieve information or updates.
+
+**Installer**
+ - [#14310](https://dev.haiku-os.org/ticket/14310) The installer now supports the user showing and selecting optional packages, when they are available on the installation medium
+
+**Launchbox**
+ - [#11237](https://dev.haiku-os.org/ticket/11237) Launchbox will now start automatically at boot, when that has been configured in its settings.
+ - [#13835](https://dev.haiku-os.org/ticket/13835) Icons of 96x96 and 128x128 pixels are now supported.
+
+**MediaPlayer**
+ - [#6628](https://dev.haiku-os.org/ticket/6628) When MediaPlayer is opened by opening a media file from Tracker, there will be a mark at the timestamp at the point where the user last stopped listening/viewing that file. 
+
+**People**
+ - [#9397](https://dev.haiku-os.org/ticket/9397) It is possible to quickly create a new Person file from Tracker, with the 'Create new...' feature
+
+**SoftwareUpdate**
+ - [#14002](https://dev.haiku-os.org/ticket/14002) When updating the system packages itself, SoftwareUpdater will now display a 'Restart' button to inform the user that a restart is necessary.
+
+**Terminal**
+ - [#15294](https://dev.haiku-os.org/ticket/15294) Support for an emulated Meta key
+
+**Tracker**
+ - [#1264](https://dev.haiku-os.org/ticket/1264) multi-range selection in Tracker and list views
+ - [#3011](https://dev.haiku-os.org/ticket/3011) Directories with a large number of files are now loaded quicker
+ - [#5126](https://dev.haiku-os.org/ticket/5126) Improvements to the Tracker API
+ - [#10529](https://dev.haiku-os.org/ticket/10529) For files that are part of a package, show information about that package in the info window.
+
+**WebPositive**
+ - [#14845](https://dev.haiku-os.org/ticket/14845) Allow selecting any font size (was previously limited to 18pt).
+
+**Command Line Utilities**
+ - [#10288](https://dev.haiku-os.org/ticket/10288) `uname` now shows which specific Haiku nightly revision is built
+ - [#12665](https://dev.haiku-os.org/ticket/12665) the built in `ftp` is now replaced with [https://ftp.netbsd.org/pub/pkgsrc/current/pkgsrc/net/tnftp/README.html tnftp]
+
+### Drivers
+
+ - The `hda` driver has had some improvements, that will allow it to work properly on more devices. If you had problem in R1/beta1, then it is worth a shot to try it again.
+ - [#9910](https://dev.haiku-os.org/ticket/9910) NVMe device support
+ - [#5056](https://dev.haiku-os.org/ticket/5056) Intel Extreme: support X4500HD
+ - [#12723](https://dev.haiku-os.org/ticket/12723) Intel Extreme: support hardware panel scaling for chipsets that support it
+ - [#13987](https://dev.haiku-os.org/ticket/13987) FreeBSD Network Drivers: upgrade network drivers to FreeBSD 11.1
+
+### Preferences
+
+ - [#10171](https://dev.haiku-os.org/ticket/10171) Support configuration of advanced mice and trackballs
```

Comment:
I think the highlight section about this is more interesting and descriptive and there is no point in linking to a specific ticket here.